### PR TITLE
BugFix: Help window formatting

### LINKED ISSUE #80: Bugs in help window
image

- Formatting for `add`, `edit` and `exit` commands are missing whitespace. Description should align with rest of the commands.
- `Open User Guide` button does not seem to be doing anything

### PR DESCRIPTION
**Pull request by RyanNgWH**
## Description
Help window currently shows different formats for different OS. This PR sets the formatting based on the system OS for a centralized look.
> `Open User Guide` button is reverted to `Copy URL` button for Linux machines as the implementation it is unsupported

Resolves #80.

### Linux
image

### MacOS
image

### Windows
image

### CODE REVIEW

**Review by dillontkh — APPROVED** [review]
Seems a little hacky, but it should do for now!

LGTM!

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -114,7 +127,14 @@ public void focus() {
     @FXML
     private void openUrl() {
         try {
-            Desktop.getDesktop().browse(new URI(USERGUIDE_URL));
+            if (System.getProperty("os.name").trim().toLowerCase().contains(OS.LINUX.toString().toLowerCase())) {
```
Comment (ruijietay) [inline src/main/java/seedu/address/ui/HelpWindow.java]: would this benefit from using the `AppUtil.isLinux()` method?

**Review by ruijietay — APPROVED** [review]
Small nit regarding the possible use of `AppUtil` methods, but other than that, LGTM!! 🥳

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (RyanNgWH) [reply src/main/java/seedu/address/ui/HelpWindow.java]: Good catch! Already made the changes

**Comment by bgopi23** [comment]
why doesnt it work in linux?

**Pull request merged**